client/src/hooks/user/newsFeedHooks.ts

```diff
@@ -0,0 +1,30 @@
+import useSWR from 'swr'
+import { AxiosResponse } from 'axios'
+
+import axios from '~/shared/lib/axios'
+import { IPost } from '~/shared/interfaces'
+
+const newsFeedHooks = () => {
+  const { data, mutate } = useSWR(
+    '/api/post',
+    () =>
+      axios
+        .get('/api/post')
+        .then((res: AxiosResponse) => res.data as IPost[])
+        .catch((error) => {
+          if (error.response.status !== 409) throw error?.response?.statusText
+        }),
+    {
+      refreshInterval: 1000,
+      revalidateOnMount: true
```

Comment:
Refreshing the data after 1000? I think it's somewhat expensive. We can use pusher if we want to implement real-time data

Reply:
Pusher is limited... do we need to implement it?